node/src/components/fetcher/tests.rs

```diff
@@ -117,7 +117,7 @@ impl reactor::Reactor<TestRng> for Reactor {
         let network = NetworkController::create_node(event_queue, rng);
 
         let (storage_config, _storage_tempdir) = storage::Config::default_for_tests();
-        let storage = Storage::new(&storage_config).unwrap();
+        let storage = Storage::new(WithDir::new(_storage_tempdir.path(), storage_config)).unwrap();
```

Comment:
Variables have a leading underscore to silence compiler warnings about them being unused.  Since `_storage_tempdir` is now used, could you remove the leading underscore please?

Reply:
The reactor constructor takes in the argument as _storage_tempdir, but I modified it accordingly